domains/kroking.is-cool.dev.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "../schemas/domain.schema.json",
+
+	"description": "Personal email",
+
+	"domain": "is-cool.dev",
+	"subdomain": "kroking",
+
+	"owner": {
+		"repo": "https://github.com/mahmoudgalalz/register",
+		"email": "ssss.aaa2002@gmail.com"
+	},
+
+	"record": {
+		"TXT":"zoho-verification=zb95553918.zmverify.zoho.com"
```

Comment:
Doesn't this need to be a nested subdomain?

Reply:
I don't know what does that mean
but i will use it like hi@kroking.is-cool.dev

Reply:
I'm not sure, read through the Zoho docs.